doc/recipe/preprocessor.rst

```diff
@@ -507,6 +529,16 @@ or alternatively:
           mask_out: sea
           fx_variables: [{'short_name': 'sftgif', 'exp': 'piControl'}]
 
+Additionally, it is possible to search across all ensembles and experiments (or any other keys)
+when specifying the fx variable, by using the ``*`` character, which is useful for some projects
+where the location of the fx files is not consistent.
+This makes it possible to search for fx files under multiple ensemble members or experiments.
+For example: ``ensemble: '*'``. Note that the ``*`` character must be quoted since ``*`` is a
+special charcter in YAML. This functionality is only supported for time invariant fx variables
+(i.e. frequency ``fx``). Note also that if multiple folders of matching fx files are found,
+ESMValTool will default to ensemble r0i0p0 if it exists and then first folder found only 
+if it does not.
+
```

Comment:
I think you can just write this paragraph once (at the top, or wherever you think it's best suited) and then just reference it via a Markdown reference, rather tha writing it three times. Up to you